packages/flutter_tools/test/general.shard/cache_test.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:collection/collection.dart' show IterableExtension;
 import 'package:file/file.dart';
+import 'package:file/local.dart' as local_fs;
```

Comment:
Looks like this is failing the forbidden_imports_test:

```
01:30 +23 -1: test/integration.shard/forbidden_imports_test.dart: no unauthorized imports of package:file/local.dart [E]                                                                               
  test/general.shard/cache_test.dart imports 'package:file/local.dart'; use 'lib/src/base/file_system.dart' instead
  package:matcher                                            fail
  test/integration.shard/forbidden_imports_test.dart 180:13  main.<fn>
  test/src/common.dart 177:18                                test.<fn>
```

Reply:
Hmm. I don't remember. What's a safe way to get access to the local file system and flutter root from there? `globals`?

Reply:
Switched to `globals.fs`.